### PR TITLE
Id transfer to CurrentApartment component and getting the current object from ApartmentService

### PR DESCRIPTION
**Pull request by Psaruklily**

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
 
   apartments: Object[] = [];
 
-  constructor(private apartmentsService: ApartmentsService) { }
+  constructor(private apartmentsService: ApartmentsService,
+    private router: Router) { }
+
+     goToApartment($event){
+       this.router.navigate(['/currentApartment']);
+       //console.log($event);
```
Comment (Oleh1033) [inline src/app/home/home.component.ts]: do you need this comment?
if "yes", for what? =)

File: src/app/current-apartment/current-apartment.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  ngOnInit(): void {
+    this.apartmentsService.currentMessage.subscribe(message => this.message = message);
+    this.currentApartment = this.apartmentsService.getApartmentById(this.message);
+    this.slides = this.currentApartment['slides'];
+    console.log(this.slides)
+      
+  }
+
+
```
Comment (Oleh1033) [inline src/app/current-apartment/current-apartment.component.ts]: empty lines

File: src/app/apartment/apartment.component.ts [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
+
+  constructor(private apartmentsService: ApartmentsService) { }
+
+  @Output() transfertoHome = new EventEmitter();
+  eventTransfer($event){
+    this.transfertoHome.emit($event);
+  }
 
   ngOnInit(): void {
+  /*   this.apartments = this.apartmentsService.getApartments();
```
Comment (Oleh1033) [inline src/app/apartment/apartment.component.ts]: comments

File: src/app/carousel/carousel.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    this.currentSlide = previous < 0 ? this.slides.length - 1 : previous;
+    console.log("previous clicked, new current slide is: ", this.currentSlide);
+  }
+
+  onNextClick() {
+    const next = this.currentSlide + 1;
+    this.currentSlide = next === this.slides.length ? 0 : next;
+    console.log("next clicked, new current slide is: ", this.currentSlide);
+  }
+
```
Comment (Oleh1033) [inline src/app/carousel/carousel.component.ts]: empty line

Comment (Psaruklily) [reply src/app/carousel/carousel.component.ts]: +

Comment (Psaruklily) [reply src/app/carousel/carousel.component.ts]: Done with it

**Pull request merged**